packages/atomic/playwright.config.ts

```diff
@@ -40,7 +40,10 @@ export default defineConfig({
   },
   webServer: process.env.CI
     ? {
-        command: 'npx http-server ./dist-storybook -p 4400',
+        command:
+          process.env.DEPLOYMENT_ENVIRONMENT === 'CDN'
+            ? 'cp -r ../headless/dist/browser ./dist-storybook/headless/ && cp -r ../bueno/dist/browser ./dist-storybook/bueno/ && npx ws -c ws.config.cjs'
+            : 'npx ws -d ./dist-storybook -p 4400',
         port: 4400,
```

Comment:
The port is specified here, in ws.config.cjs, and in 'npx ws -d ./dist-storybook -p 4400'; is it necessary?

Reply:
same